src/lib.rs

```diff
@@ -1,5 +1,59 @@
 //! Winit is a cross-platform window creation and event loop management library.
 //!
+//! # Usage
+//!
+//! `winit` can be added to `Cargo.toml` as a dependency. It can be added via `cargo add`.
+//!
+//! ```bash
+//! $ cargo add winit
+//! ```
+//!
+//! To only enable the X11 backend on Free Unix[^unix] systems, disable default features
+//! and enable the `x11` feature.
+//!
+//! ```bash
+//! $ cargo add winit --no-default-features --features x11
+//! ```
+//!
+//! To only enable the Wayland backend on Free Unix systems, disable default features
+//! and enable the `wayland` feature.
+//!
+//! ```bash
+//! $ cargo add winit --no-default-features --features wayland
+//! ```
+//!
+//! These features have no effect on systems that are not Free Unix.
```

Comment:
Should this spend a few more words explaining that:

- Disabling default features on other systems/targets has no effect, since they only have one "implementation" that is unconditionally compiled in;
- Enabling `x11`/`wayland` is a no-op on non-Free-Unix systems, i.e. enabling it while also building for Windows or Android doesn't suddenly break the build?